addons/jaws-esr/bootstrap.js

```diff
@@ -9,13 +9,21 @@
 
 ChromeUtils.import("resource://gre/modules/Services.jsm");
 ChromeUtils.import("resource://gre/modules/AppConstants.jsm");
+ChromeUtils.import("resource://gre/modules/XPCOMUtils.jsm");
```

Comment:
Looks like `ChromeUtils.defineModuleGetter` was around in 60, so you could probably avoid this XPCOMUtils.jsm import and just use that below if you want... not important though.